### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by mreid-moz**
Once-over through all the scala code to clean up style and formatting.

### CODE REVIEW

**Comment by maurodoglio** [comment]
Did you run a linter to fix all these style issues?

**Comment by mreid-moz** [comment]
Rebased!

No, wasn't a linter. I looked mostly at warnings from IntelliJ's Scala IDE and fixed things manually.

File: src/main/scala/com/mozilla/telemetry/histograms/Histogram.scala [hunk 1]
```diff
@@ -99,12 +99,12 @@ object Histograms {
     }
 
     // Histograms are considered to be immutable so it's OK to merge their definitions
-    parsed.flatMap(_._2).toMap
```
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/histograms/Histogram.scala]: What's the purpose of this change?

File: src/main/scala/com/mozilla/telemetry/streams/Longitudinal.scala [hunk 2]
```diff
@@ -568,14 +567,18 @@ case class Longitudinal() extends DerivedStream {
       record
     }
 
-    val fieldSchema = schema.getField(avroField).schema().getTypes()(1).getElementType()
```
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/streams/Longitudinal.scala]: this doesn't look like a style fix. maybe it's worth a separate commit?

Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/streams/Longitudinal.scala]: nevemind, I just noticed the addition below 😸

**Comment by maurodoglio** [comment]
🚢 it !

File: src/main/scala/com/mozilla/telemetry/histograms/Histogram.scala (hunk 1 above)
Comment (mreid-moz) [inline src/main/scala/com/mozilla/telemetry/histograms/Histogram.scala]: It's already a map, so the `toMap` doesn't do anything.

**Comment by vitillo** [comment]
Nice job, thanks @mreid-moz!
